fission-web-server/library/Fission/Web/Server/Email/SendInBlue/TemplateOptions/Types.hs

```diff
@@ -1,18 +0,0 @@
-module Fission.Web.Server.Email.SendInBlue.TemplateOptions.Types (TemplateOptions(..)) where
-
-import           Servant.Client
-
-import           Fission.Prelude
-
-import           Fission.User.Username.Types
-
-data TemplateOptions = TemplateOptions
-  { verifyLink :: BaseUrl
-  , username   :: Username
-  }
-
-instance ToJSON TemplateOptions where
-  toJSON TemplateOptions { verifyLink, username } =
-    Object [ ("VERIFY_LINK", toJSON verifyLink )
-           , ("USERNAME",    toJSON username )
-           ]
```

Comment:
I guess we need to change some stuff in SiB?

Reply:
Whoops, no it's below just renamed, gotcha. Odd that sometimes git can't figure this stuff out

Reply:
> I guess we need to change some stuff in SiB?

There is changes for some things in SIB, but not for the existing stuff: We needed to add the new recovery email template, but I've already done that as my pizza instance is hooked up to "production" SIB.